tools/shared.py

```diff
@@ -1372,61 +1371,6 @@ def add_emscripten_metadata(js_file, wasm_file):
       f.write(contents)
       f.write(orig[8:])
 
-  @staticmethod
-  def add_dylink_section(wasm_file, needed_dynlibs):
```

Comment:
I have a PR in progress that modifies this method to work with the wasm backend.   So can you maybe leave it in for now?

Reply:
Ok, I'll restore `add_dylink_section`.